src/component/RoadmapCanvas.js

```diff
@@ -28,6 +30,12 @@ export class RoadmapCanvas {
     if (style.barHeight !== undefined) {
       this.style.barHeight = parseInt(style.barHeight);
     }
+    if (style.timeFormat !== undefined) {
```

Comment:
hum... It's is not wrong. But bit ugly...

I think, We should create `RoadmapOption`.
What dot you think?